README.md

```diff
@@ -109,3 +109,29 @@ To register pre-commit check to run Ktlint format:
 ```
 ./gradlew ktlintApplyToIdea addKtlintCheckGitPreCommitHook
 ```
+
+
+#### Send notifications locally
+To help test notification events locally we can send events to localstack to replicate what NOMIS would do -
+##### Step 1 - Start Orchestration service on local
+
+##### Step 2 - Install awscli (if not already installed)
+```
+brew install awscli
+```
+
+##### Step 3 - configure aws with dummy values (if not already configured)
+```
+aws configure
+```
+Put any dummy value for AWS_ACCESS_KEY and AWS_SECRET_KEY and eu-west-2 as default region.
+
+##### Step 4 - Send a message to the application queue. The below is a non-assciation event for prisoners G0026GC and A8713DY. Replace the prisoner numbers with your test values.
+```
+aws sqs --endpoint-url=http://localhost:4566 send-message --queue-url=http://localhost:4566/000000000000/sqs_hmpps_prison_visits_event_queue  --message-body '{"Type":"Notification", "Message": "{\"eventType\":\"non-associations.created\",\"additionalInformation\":{\"nsPrisonerNumber2\":\"G0026GC\",\"nsPrisonerNumber1\":\"A8713DY\"}}", "MessageId": "60945f6e-f146-418e-a8e9-1de858d41faa"}'
```

Comment:
I found that my `queue-url` ended with `/hmpps_prison_visits_event_queue` rather than `/sqs_hmpps_prison_visits_event_queue` - don't know whether example should be changed or just something peculiar with my set up 🤷‍♂️

Reply:
Been experimenting and if you break up the command onto separate lines like this it is a bit easier to paste into shell and edit before pressing enter:
```bash
aws sqs send-message \
  --endpoint-url=http://localhost:4566 \
  --queue-url=http://localhost:4566/000000000000/hmpps_prison_visits_event_queue \
  --message-body \
    '{"Type":"Notification", "Message": "{\"eventType\":\"non-associations.created\",\"additionalInformation\":{\"nsPrisonerNumber2\":\"G4206GT\",\"nsPrisonerNumber1\":\"G4216GE\"}}", "MessageId": "123"}'
```

Reply:
I have checked the application_dev.yml and it is sqs_hmpps_prison_visits_event_queue. I have also added some instructions on how to find the value.
Updated README with the command you have added on here.